dev/io.openliberty.checkpoint_fat_security_oidc_social/bnd.bnd

```diff
@@ -21,7 +21,8 @@ src: \
 fat.project: true
 
 tested.features: jaxrs-2.0, el-3.0, restfulwsclient-3.0, restfulws-3.0, appsecurity-4.0, cdi-2.0, expressionlanguage-4.0, pages-3.0, \
-                 jsonp-1.1, jaxrsclient-2.1, restfulws-3.1, appsecurity-5.0, pages-3.1, jaspic-1.1, appauthentication-2.0, appauthentication-3.0, xmlbinding-3.0, checkpoint
+                 jsonp-1.1, jaxrsclient-2.1, restfulws-3.1, appsecurity-5.0, pages-3.1, jaspic-1.1, appauthentication-2.0, appauthentication-3.0, xmlbinding-3.0, checkpoint, \
```

Comment:
Just a small typo that appsecuirty-6.0 should be appsecurity-6.0

Reply:
Done, thanks!